### ISSUE TITLE
Cannot find module './instagramBasicApi' in production (ubuntu)

### ISSUE THREAD

**Issue by SvenWesterlaken** [webvibe-io/strapi-plugin-instagram#12]
Locally this does not cause any errors, however when starting in production on ubuntu 22.04:

```
Error: Could not load js config file /home/github/strapi/node_modules/@webvibe-io/strapi-plugin-instagram/strapi-server.js: Cannot find module './instagramBasicApi'
Require stack:
- /home/github/strapi/node_modules/@webvibe-io/strapi-plugin-instagram/server/services/index.js
- /home/github/strapi/node_modules/@webvibe-io/strapi-plugin-instagram/server/index.js
- /home/github/strapi/node_modules/@webvibe-io/strapi-plugin-instagram/strapi-server.js
- /home/github/strapi/node_modules/@strapi/utils/lib/import-default.js
- /home/github/strapi/node_modules/@strapi/utils/lib/index.js
- /home/github/strapi/node_modules/@strapi/strapi/lib/Strapi.js
- /home/github/strapi/node_modules/@strapi/strapi/lib/index.js
- /home/github/strapi/node_modules/@strapi/strapi/lib/commands/transfer/utils.js
- /home/github/strapi/node_modules/@strapi/strapi/bin/strapi.js
    at loadJsFile (/home/github/strapi/node_modules/@strapi/strapi/lib/core/app-configuration/load-config-file.js:18:11)
    at loadFile (/home/github/strapi/node_modules/@strapi/strapi/lib/core/app-configuration/load-config-file.js:35:14)
    at Object.loadPlugins (/home/github/strapi/node_modules/@strapi/strapi/lib/core/loaders/plugins/index.js:103:26)
    at async Strapi.loadPlugins (/home/github/strapi/node_modules/@strapi/strapi/lib/Strapi.js:341:5)
    at async Promise.all (index 2)
    at async Strapi.register (/home/github/strapi/node_modules/@strapi/strapi/lib/Strapi.js:377:5)
    at async Strapi.load (/home/github/strapi/node_modules/@strapi/strapi/lib/Strapi.js:487:5)
    at async Strapi.start (/home/github/strapi/node_modules/@strapi/strapi/lib/Strapi.js:217:9)
```

Might have to do with the import not having an extension and cannot be resolved on certain systems?

Note that I haven't done anything yet with this strapi instance; clean install without configuring anything.

Node Version: 19.5.0
Strapi Version: 4.6.0

**Comment by SvenWesterlaken** [comment]
Same error as https://github.com/webvibe-io/strapi-plugin-instagram/issues/11

**Comment by kepes** [comment]
I don't understand why Strapi can't load `instagramBasicApi.js because the file is there. Maybe there is some issue with Node version 19. As I can see Strapi don't support this version yet. 

https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/installation/cli.html

**Comment by kepes** [comment]
I figured out there were some camel case issue. I fixed it in [version 1.2.3](https://github.com/webvibe-io/strapi-plugin-instagram/releases/tag/v1.2.3) just upgrade package with `npm upgrade @webvibe-io/strapi-plugin-instagram`

**Comment by SvenWesterlaken** [comment]
> I figured out there were some camel case issue. I fixed it in [version 1.2.3](https://github.com/webvibe-io/strapi-plugin-instagram/releases/tag/v1.2.3) just upgrade package with `npm upgrade @webvibe-io/strapi-plugin-instagram`

That would make sense that it fails on Ubuntu. Windows & Mac do correct this, but Ubuntu/Linux doesn't. Thanks for fixing it @kepes!
